### PR TITLE
feat: store and reload HMPB package

### PR DESCRIPTION
**Pull request by eventualbuddha**
This changes the API to expose a more generic `/election` endpoint for manipulating the definition. We now store the decoded image data for uploaded HMPB templates and reload them on startup.

Refs https://github.com/votingworks/bsd/pull/69

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -158,14 +192,40 @@ export default class Store {
     }
 
     if (resetDB) {
```
Comment (benadida) [inline src/store.ts:194]: do we need to add removal of the template files on reset, too?

Comment (eventualbuddha) [reply src/store.ts:194]: Good catch.

**Comment by eventualbuddha** [comment]
Storing just the PDFs would be simpler. I’ll look into it.

**Comment by eventualbuddha** [comment]
I updated to store the PDFs in the db.

File: src/store.ts [hunk 2]
```diff
@@ -153,17 +175,43 @@ export default class Store {
    * Initializes the database by destroying the existing one if needed.
    */
   public async init(resetDB = false): Promise<void> {
-    if (this.db) {
-      await this.dbDestroy()
-    }
+    if (resetDB || !this.db) {
```
Comment (eventualbuddha) [inline src/store.ts:178]: I changed the logic of how we reset the db. Without this, in tests `init` would get called once on creating the `memoryStore` and again in `start`, which would clear everything out. This makes it require a flag to actually destroy the db.

**Pull request merged**